packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -175,7 +172,18 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
 
   if (organization.activeDomain) {
     await enableUsageStats(billingLeadersIds, orgId)
-    await sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader)
+    await Promise.all([
+      sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader),
+      billingLeaders.map((billingLeader) =>
+        sendTeamsLimitEmail({
+          user: billingLeader,
+          orgId,
+          orgName,
+          emailType: 'thirtyDayWarning'
+        })
+      ),
+      scheduleTeamLimitsJobs(scheduledLockAt, orgId)
```

Comment:
-1 I missed this before, but this seems to be the wrong place. This limit is about teams limit, so why would we only send this when an organization has an `activeDomain` (i.e. qualifies for enterprise)?

In addition it seems to me like `scheduleTeamLimitsJobs` and `scheduleJobs` below should be merged.

Reply:
I think we just need to move this check above everything.
Changed it here: https://github.com/ParabolInc/parabol/pull/7682/files/d2b3b5e426ec2537f2352ce3af49f4ec5d42db00#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR144

Reply:
@igorlesnenko you're thinking I add `if (!organization.activeDomain) return` to line 144?

Reply:
@nickoferrall yes, basically do not run any limits checks if organization does not have a domain.

Reply:
You did not answer my question of **why** we want to have the team limit only for organizations with an `activeDomain`.

Reply:
I don't see why we'd need to check `activeDomain` as, according to [this message](https://parabol.slack.com/archives/C042E8PGPD4/p1675267220136159?thread_ts=1675266998.778289&cid=C042E8PGPD4) 🔒, we have a Team tier team that doesn't have an active domain. 

I've removed this check, but @igorlesnenko please let me know if I'm missing something

Reply:
@nickoferrall The check was added because of 2 reasons:

1) Currently, it is not possible to show usage stats for organizations without a domain.

I think for now we need to keep this check at least in `enableUsageStats`.

2) We want to exclude deals that is in progress on HubSpot. We use the domain to check this.

@tghanken is it possible to determine if the deal is in progress on HubSpot if we don't know an organization domain? In this case, can we assume that the deal is not in progress and safely trigger the limit warnings?

Reply:
Hmm but if we just add the check to `enableUsageStats`, wouldn't `sendWebsiteNotifications` still execute, which would trigger a notification telling the user to check their usage (see https://github.com/ParabolInc/parabol/pull/7601)?

Perhaps a temporary solution could be adding the `if (!organization.activeDomain) return` check again, but creating a new issue to either a) enable usage stats for teams without an active domain or b) show a different notification if there isn't an active domain?

Also, feel free to respond after your holiday!

Reply:
> is it possible to determine if the deal is in progress on HubSpot if we don't know an organization domain?

Organizations with a null `activeDomain` field will not have a corresponding company in HubSpot, so there is effectively no way a deal can automatically be created for an organization with no domain.  It feels like a safe assumption that any org with a null `activeDomain` will not have a deal in progress.  The only exception would be if a user on the domain initiates contact with our sales team.  I'll check with the team if we have a process for handling inbound sales requests from users on null domains.

> Perhaps a temporary solution could be adding the if (!organization.activeDomain) return check again, but creating a new issue to either a) enable usage stats for teams without an active domain or b) show a different notification if there isn't an active domain?

I think this is the way to go.  The safe, fast approach is to keep the check, and we can optimize for the small tail of qualified null `activeDomain` orgs after the fact.  Right now these orgs are a VERY small subset of our qualified orgs.

Reply:
> Perhaps a temporary solution could be adding the if (!organization.activeDomain) return check again, but creating a new issue to either a) enable usage stats for teams without an active domain or b) show a different notification if there isn't an active domain?

Yes, I agree. We can run experiment only on organizations with active domain for now and improve later.

In the next new issue we also need a way to check for an active deal on HubSpot for orgs without a domain.

Reply:
Sounds good. I've created an issue here https://github.com/ParabolInc/parabol/issues/7723 and pushed an update with a comment linking to the issue